spec/classes/director_spec.rb

```diff
@@ -9,11 +9,11 @@
 
       expected_packages = []
 
-      case facts[:osfamily]
+      case facts[:os][:family]
```

Comment:
did you check if those tests are actually executed? Only the first key in `facts` is a symbol. So it needs to be:
```suggestion
      case facts[:os]['family']
```

however it could be that rspec-puppet-facts turns that into a string and both ways work.

Reply:
Good catch!

Before:
```
250 examples, 0 failures
```

After:
```
286 examples, 0 failures
```